backend/services/task.service.js

```
@@ -0,0 +1,511 @@
+  /**
+   * Merge task operations
+   */
+  async merge(taskId, operation = 'toBase', options = {}) {
+    const { githubToken } = options;
+    
+    const task = await this.repository.findById(taskId);
+    const project = await this.repository.getProject(taskId);
+    
+    switch (operation) {
+      case 'toBase':
+        const result = await this.gitOps.mergeToBase(task, project, githubToken);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)